doc/api/http.md

```diff
@@ -1288,6 +1288,13 @@ the server, then sockets are destroyed when they time out. If a handler is
 assigned to the request, the response, or the server's `'timeout'` events,
 timed out sockets must be handled explicitly.
 
+### response.aborted
+<!-- YAML
+added: REPLACEME
+-->
+
+If a request has been aborted.
```

Comment:
Is it a typo or confusing API? **response**.aborted means **request** is aborted?